plugins.json

```diff
@@ -4085,6 +4085,187 @@
     },
     "updated_at": "2019-06-11T19:41:07Z"
   },
+  {
+    "homepage_url": "https://github.com/mattermost/mattermost-plugin-calls/",
+    "icon_data": "",
+    "download_url": "https://plugins-store.test.mattermost.com/release/mattermost-plugin-calls-v0.14.0.tar.gz",
+    "release_notes_url": "https://github.com/mattermost/mattermost-plugin-calls/releases/tag/v0.14.0",
+    "hosting": "",
+    "author_type": "mattermost",
+    "release_stage": "production",
+    "enterprise": false,
+    "signature": "iQIzBAABCAAdFiEExViBuA9p6GO4WtXR0bVLR6XO/sQFAmQJJXgACgkQ0bVLR6XO/sSaKQ/9El5Ww2F9HSvm0Z2u5T+0FZjcN8zwXmQGPX6QpfDfW2m88jmozyEK7GyXJBuoUJ6CGkKjA5vmfeTpZhMDMyaszHHdvrWbREjoquJUzZgejCQbBAO9Xzoy47SDWPSJ6/pG7kBSRLiGHR7oMpLrMqhAmGMAvrZONieVsnIEr6aFUtpXm/ItCAlWc7AwgsgkffbvD0cA1bkKlETYPzkW81pOq8L02PiiMdddFXVwmy/D2iJXXlYtQ+6pchURJmiYBglCO0JsXo0oEHF+VYKIHLRT2UhTywiOSBMzDfN3A6tz9PXKEwnwYXKf4ydEPYzjXimbhgabThDsPGAYa/bP+4MY39oV1dmC+1tsrKMCLOMW+haRbuCNI2+5ObBzNiPY1MadzZrnZXCAPzpeSSHywk76rekqi1TXsBeLknGmwEkEKDdOsp9AQq5pifyuGjZ9sFHr9d35Oo4xjtFAIejqd9cx3a4fWKh/qhLTWt2ImG6Gf/rk5psHO+vKoS67c0Iag2ecQRrHYEEhsTHzlfwM8jy5CBIRqjLnodyGZrm8ho+Sxbyq+Mqst7xZTIUt3x3oG0c71ed3iRimVz5Om6n8Rs1Nycpymq9IVitOUKXV5GX3Eyhot4KTMaHWozaNBxhwX4hDERKhMA/nougvVQQ/3vN953pTVWoz0dFFb5eVqN1trWg=",
+    "repo_name": "mattermost-plugin-calls",
+    "manifest": {
+      "id": "com.mattermost.calls",
+      "name": "Calls",
+      "description": "Integrates real-time voice communication in Mattermost",
+      "homepage_url": "https://github.com/mattermost/mattermost-plugin-calls/",
+      "support_url": "https://github.com/mattermost/mattermost-plugin-calls/issues",
+      "release_notes_url": "https://github.com/mattermost/mattermost-plugin-calls/releases/tag/v0.14.0",
+      "version": "0.14.0",
+      "min_server_version": "7.6.0",
+      "server": {
+        "executables": {
+          "darwin-amd64": "server/dist/plugin-darwin-amd64",
+          "darwin-arm64": "server/dist/plugin-darwin-arm64",
+          "freebsd-amd64": "server/dist/plugin-freebsd-amd64",
+          "linux-amd64": "server/dist/plugin-linux-amd64",
+          "linux-arm64": "server/dist/plugin-linux-arm64",
+          "openbsd-amd64": "server/dist/plugin-openbsd-amd64"
+        },
```

Comment:
At the same time, it seems these these are all compiled, so I would expect them to be in the individually-built binaries referenced in the above comment. I see something related to calls in matterbuild's scripts. I'm not sure if this is still occurring in the most up-to-date matterbuild though

https://github.com/mattermost/matterbuild/blob/59f5f2df3571c981871ed21cf78a420d1934e539/server/plugin_release.go#L411-L416

Reply:
Looks like the "individually building" script needs to be updated in general, to support more platforms. The corner case for Calls is right next to that code.

Reply:
Not sure what you mean? The matterbuild code builds and verifies the package, and that's what the generator parsed and output into the `plugins.json`.

Reply:
In the comment above https://github.com/mattermost/mattermost-marketplace/pull/347#discussion_r1131493026, the `darwin-amd64` and `windows-amd64`-specific bundles are missing in the new marketplace entry. This is because matterbuild is not individually packaging those "single-architecture" bundles.

I'm saying that:
- The general case of packaging these this way should take account making single-architecture bundles for all 6 architectures we now support, instead of just the original 3 we supported
- and the corner case for calls also should do something similar, since it is only packaging one single-architecture bundle at the moment

Just so I understand clearly, the calls plugin indeed runs on the above 6 architectures? I think at some point this wasn't the case so just checking.

The single-architecture bundle is an optimization we made for spinning up Cloud instances quicker (much smaller bundle to download), so I don't think the above points are required changes. The multi-architecture bundles are bigger, but the benefits are probably negligible for on-prem deployments.

Reply:
@mickmister Calls is only officially supported on Linux (amd64). Any other target was either added by community contributors (open/freebsd) and darwin is used for development only so it shouldn't get shipped in any official build.